samples/XAMLCatalog/XAMLCatalog/UIKitControls/UIButtonWithControlsViewController.h

```diff
@@ -0,0 +1,27 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "MenuTableViewController.h"
+
+@interface UIButtonWithControlsViewController : UIViewController <UITextFieldDelegate>
+
+@property (nonatomic, readonly) UIButton* button;
+@property (nonatomic, readonly) UITextField* textButtonNormal;
```

Comment:
In view of the other member names,, it would probably be textTitleNormal (similar to sliderTitleColorNormal).

---
In reply to: [97700191](https://github.com/Microsoft/WinObjC/pull/1771#discussion_r97700191) [](ancestors = 97700191)